styles/main.css

```diff
@@ -2,22 +2,21 @@
 
 body {
     background-color: #9ad89a; 
+    font-family: 'Open Sans', sans-serif;
 }
 
 h1 {
     color: #6e309e; 
     background-image: url("../images/gradient.png"); 
     background-repeat: repeat-x; 
     font-family: 'Proza Libre', sans-serif;
-    font-family: 'Open Sans', sans-serif;

```

Comment:
Why are you setting two fonts here?  Only one will be used.  If you're going for a back-up font you could use: `font-family: 'Proza Libre', 'Open Sans', sans-serif;`